plugins/wazuh_check_updates/public/components/upToDateStatus.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHealth,
+  EuiIconTip,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import { useAvailableUpdates } from '../hooks';
+import { formatUIDate, getCurrentAvailableUpdate } from '../utils';
+import { Update } from '../../common/types';
+
+export interface UpToDateStatusProps {
+  setCurrentUpdate: (currentUpdate?: Update) => void;
+}
+
+export const UpToDateStatus = ({ setCurrentUpdate }: UpToDateStatusProps) => {
+  const { availableUpdates, isLoading, refreshAvailableUpdates, error } = useAvailableUpdates();
+
+  const handleOnClick = async () => {
+    await refreshAvailableUpdates(true);
+  };
+
+  const currentUpdate = getCurrentAvailableUpdate(availableUpdates);
+
+  setCurrentUpdate(currentUpdate);
```

Comment:
**issue**: This is called in each render of the component.

Reply:
Added useEffect